bundles/org.openhab.binding.airq/src/main/java/org/openhab/binding/airq/internal/AirqHandler.java

```diff
@@ -439,65 +439,70 @@ public void dispose() {
+                processType(decObj, "co", "co", "pair");
+                processType(decObj, "co2", "co2", "pairPPM");
+                processType(decObj, "dewpt", "dewpt", "pair");
+                processType(decObj, "h2s", "h2s", "pair");
+                processType(decObj, "humidity", "humidityRelative", "pair");
+                processType(decObj, "humidity_abs", "humidityAbsolute", "pair");
+                processType(decObj, "no2", "no2", "pair");
+                processType(decObj, "o3", "o3", "pair");
+                processType(decObj, "oxygen", "o2", "pair");
+                processType(decObj, "pm1", "fineDustConc01", "pair");
+                processType(decObj, "pm2_5", "fineDustConc02_5", "pair");
+                processType(decObj, "pm10", "fineDustConc10", "pair");
+                processType(decObj, "pressure", "pressure", "pair");
+                processType(decObj, "so2", "so2", "pair");
+                processType(decObj, "sound", "sound", "pairDB");
+                processType(decObj, "temperature", "temperature", "pair");
+                // We have two places where the Device ID is delivered: with the measurement data and
+                // with the configuration.
+                // We take the info from the configuration and show it as a property, so we don't need
+                // something like processType(decObj, "DeviceID", "DeviceID", "string") at this moment. We leave
+                // this as a reminder in case for some reason it will be needed in future, e.g. when an air-Q
+                // device also sends data from other devices (then with another Device ID)
+                processType(decObj, "Status", "status", "string");
+                processType(decObj, "TypPS", "avgFineDustSize", "number");
+                processType(decObj, "dCO2dt", "dCO2dt", "number");
+                processType(decObj, "dHdt", "dHdt", "number");
+                processType(decObj, "door_event", "doorEvent", "number");
+                processType(decObj, "health", "healthIndex", "index");
+                processType(decObj, "health", "health", "number");
+                processType(decObj, "measuretime", "measureTime", "number");
+                processType(decObj, "performance", "performanceIndex", "index");
+                processType(decObj, "performance", "performance", "number");
+                processType(decObj, "timestamp", "timestamp", "datetime");
+                processType(decObj, "uptime", "uptime", "numberTimePeriod");
+                processType(decObj, "tvoc", "tvoc", "pairPPB");
+
+                updateStatus(ThingStatus.ONLINE);
+            } catch (JsonSyntaxException e) {
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
+                        "Syntax error while parsing response from device: " + e.getMessage());
+                logger.error("Parse error in response: {}", jsonAnswer);
```

Comment:
High level logging is already performed by setting the thingstate. When verbose message contents are logged, the trace level should be used.

```suggestion
                logger.trace("Parse error in response: {}", jsonAnswer);
```